examples/template/README.rst

```diff
@@ -0,0 +1,12 @@
+
+==================================================================
+                        tmt test template
+==================================================================
+
+When creating a new ``tmt`` test it is recommended to use the test
+template provided in this directory. In order to be able to use it
```

Comment:
How about a link to https://tmt.readthedocs.io/en/stable/guide.html#custom-templates ? 
Otherwise it is confusing why files are linked as .j2 to some directory which might not yet exist.